src/fastman/commands/misc.py

```
@@ -0,0 +1,654 @@
+
+# Install dependencies
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy application
+COPY . .
+
+# Run migrations and start server
+CMD ["sh", "-c", "alembic upgrade head && uvicorn app.main:app --host 0.0.0.0 --port 8000"]
+'''
+            PathManager.write_file(dockerfile, content)
```

Ruff (0.14.14)

[error] Undefined name `PathManager`

(F821)